tests/foreman/ui/test_host.py

```diff
@@ -1535,8 +1588,28 @@ def test_global_registration_token_restriction():
         1. open the global registration form and generate the curl token
         2. use that curl token to execute other api calls e.g. GET /hosts, /users
 
-    :CaseAutomation: ManualOnly
     """
+    client = rhel7_contenthost
+    with session:
+        cmd = session.host.get_register_command(
+            {
+                'general.capsule': module_proxy.name,
+                'general.operating_system': module_os.title,
+                'advanced.activation_keys': module_activation_key.name,
+                'general.insecure': True,
+            }
+        )
+
+    pattern = re.compile("Authorization.*(?=')")
+    auth_header = re.search(pattern, cmd).group()
+
+    # build curl
+    curl_users = f'curl -X GET -k -H {auth_header} -i https://{settings.server.hostname}/api/users/'
```

Comment:
Can't we use default_sat fixture here?
```suggestion
    curl_users = f'curl -X GET -k -H {auth_header} -i {default_sat.url}/api/users/'
```

Reply:
thanks modified pr bit late so missed it